packages/react-shared-contexts/README.md

```diff
@@ -0,0 +1,8 @@
+# @fluentui/react-shared-contexts
+
+**Shared contexts for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
```

Comment:
Are these for northstar too? If so we should probably update this to something like:
```
**Shared contexts for [`@fluentui/react`](v8 doc link) and [`@fluentui/react-northstar`](v0 doc link)**
```

Reply:
It's not going to be used for northstar right now. I'd prefer to not add that until needed.